src/test/java/com/bytezone/dm3270/TerminalClientTest.java

```diff
@@ -48,11 +24,33 @@
 import us.abstracta.wiresham.Flow;
 import us.abstracta.wiresham.VirtualTcpService;
 
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.awt.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.security.GeneralSecurityException;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.when;
+
 @RunWith(MockitoJUnitRunner.class)
 public class TerminalClientTest {
 
   private static final Logger LOG = LoggerFactory.getLogger(TerminalClientTest.class);
   private static final int TERMINAL_MODEL_TYPE_TWO = 2;
+  private static final int TERMINAL_MODEL_TYPE_THREE = 2;
```

Comment:
Is this right? Because it says model type three but the value is 2, if the value 2 works then you might just use mode type two.

Reply:
my bad, it's a mistake, corrected in c5a9736480f0d385585a1a645a20fca267953310